.gitignore

```diff
@@ -76,6 +76,7 @@ Thumbs.db
 /saves/
 /screenshots/
 /logs/
+/configs/
```

Comment:
This change has already been made in #3227 - if you think it is better here, I shall revert the change in that PR.